exec/vm.go

```diff
@@ -80,12 +80,26 @@ func NewVM(module *wasm.Module) (*VM, error) {
 		copy(vm.memory, module.LinearMemoryIndexSpace[0])
 	}
 
-	vm.compiledFuncs = make([]compiledFunction, len(module.FunctionIndexSpace))
+	vm.funcs = make([]function, len(module.FunctionIndexSpace))
 	vm.globals = make([]uint64, len(module.GlobalIndexSpace))
 	vm.newFuncTable()
 	vm.module = module
 
+	nNatives := 0
 	for i, fn := range module.FunctionIndexSpace {
+		// Skip native methods as they need not be
+		// disassembled; simply add them at the end
+		// of the `funcs` array as is, as specified
+		// in the spec.
```

Comment:
could you add an explicit pointer to the specs, or the name of the relevant section ?